src/components/NavBar/NavBar.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import NavBar from './NavBar.component';
+
+describe('<NavBar /> component', () => {
```

Comment:
Nice, very complete tests

Reply:
Thank you!

Ehhh , for the test coverage the .gitignore omit those files, I though you were encourage to generate the report. I don't know whether code sandbox can generate it or not. Local I did just yarm test coverage and it generates all the info. Will see how can I provide that info through next challenges.